hoodie-spark/src/test/scala/DataSourceTest.scala

```diff
@@ -227,7 +227,7 @@ class DataSourceTest extends AssertionsForJUnit {
     val f2 = Future {
       inputDF1.write.mode(SaveMode.Append).json(sourcePath)
       // wait for spark streaming to process one microbatch
-      Thread.sleep(3000)
+      Thread.sleep(10000)
```

Comment:
the test is currently disabled? what does 10s specifically fix? I think this test is better written as a - poll for commits to show up, with a large timeout value like 1 minute.. then it ll only fail if even a large timeout does not help it pass

Reply:
Reverted this